source/org/zfin/profile/service/ProfileService.java

```diff
@@ -527,25 +527,30 @@ public Person createPerson(Person person) {
 
         person.setEmailPrivacyPreference(getDefaultEmailPrivacyPreference());
 
-        AccountInfo accountInfo = new AccountInfo();
         String login = person.getPutativeLoginName();
         if (StringUtils.isEmpty(login)) {
             login = person.getEmail();
         }
+
+        person.setDeceased(false);
+        person.setUrl(processUrl(person.getUrl()));
+
+        HibernateUtil.currentSession().save(person);
```

Comment:
Do you have to save the person record twice in this method?

Reply:
Good point. I removed it.